pages/madewithcode.jsx

```diff
@@ -0,0 +1,82 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+var OutboundLink = require('react-ga').OutboundLink;
+
+var config = require('../lib/config');
+
+var Intro = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <h1>Made with Code</h1>
+        <section className="intro">
+          <Illustration
+          width={200} height={355}
+          src1x="/img/pages/madewithcode/logo_mwc.png"
+          alt="Woman training a young man on a computer"
+          className="content-first">
+            <h2>Welcome! Mozilla is happy to partner with Google's Made with Code initiative to inspire girls to get creative with code. Below are three fun activities to help you create your own webpages by writing and remixing HTML.</h2>
+          </Illustration>
+        </section>
+      </div>
+    );
+  }
+});
+
+var MadeWithCode = React.createClass({
+  statics: {
+    pageTitle: 'Made with Code',
+    pageClassName: 'madewithcode-landing'
+  },
+  curriculum: [
+    {
+      title: "",
+      activities: [
+        {
+          title: "Let's Make a Meme",
+          image1x: "/img/pages/madewithcode/doge.jpg",
+          image2x: "/img/pages/madewithcode/doge@2x.jpg",
+          subtitle: "Understanding remixing",
+          description: "A “meme” is something that’s shared on the internet, and it usually has three parts: a picture, a caption, and a dash of humor. In this activity, you’ll make your own meme using a bit of HTML and your own creativity.",
```

Comment:
nope, we are being inconsistent - I think we should stick with pretty quotes though